### PR TITLE
Conversion to single page application (Electron, webpack-dev-server).

### PR DESCRIPTION
**Pull request by stephenshank**
Note that the development and production environments have changed. Development now takes place with the webpack-dev-server. The production version will now be served by express. This also adds functionality for Electron. See the updated README.md.

This was a major overhaul that caused plenty of bugs. I've managed to fix most that I've caught, but some may persist.

### CODE REVIEW

**Comment by stevenweaver** [comment]
Dear @stephenshank, 

Has the library version (what is currently in use) been tested at all?

Best,
Steven

**Comment by stephenshank** [comment]
Dear @stevenweaver,

Using `yarn link` I tested in my Datamonkey development instance, and things appeared normal.

Best,
Stephen

**Comment by stephenshank** [comment]
Just in case it's helpful, there were some definite issues with yarn. The `yarn.lock` file had a merge conflict, so I simply cleared out `node_modules` and regenerated it while performing the merge. Also, when trying to get this to run on my personal computer, I had to run `brew upgrade yarn`.

File: src/jsx/absrel.jsx [hunk 1]
```diff
@@ -513,7 +532,7 @@ var BSREL = React.createClass({
             </div>
           </div>
         </div>
-
+        <div id="tooltip_container">This is my tooltip.</div>
```
Comment (stevenweaver) [inline src/jsx/absrel.jsx]: Doesn't seem informative.

**Pull request merged**